kuksa-client/kuksa_client/__main__.py

```diff
@@ -527,11 +511,35 @@ def connect(self):
             if self.commThread is not None:
                 self.commThread.stop()
                 self.commThread = None
-        config = {'ip': self.serverIP,
-                  'port': self.serverPort,
-                  'insecure': self.insecure,
-                  'protocol': self.serverProtocol
-                  }
+
+        # Check we have a valid server URI
+        srv = urlparse(self.server)
+        config = {"port": 55555, "insecure": True}
+
+        if srv.scheme in ["grpc", "grpcs"]:
+            config["protocol"] = "grpc"
+        elif srv.scheme in ["ws", "wss"]:
+            config["protocol"] = "ws"
+        else:
+            print(f"Invalid server URI. Unsupported protocol: {srv.scheme} ")
+            return
+
+        if srv.port is not None:
+            config["port"] = srv.port
+
+        if srv.scheme in ["grpcs", "wss"]:
```

Comment:
As default value for `config["insecure"]` may change in the future, shall we also set it explicitly if "grpc" or "ws" is used, i.e. add an else branch where we set it True?